wallet/src/de/schildbach/wallet/payments/SendCoinsTaskRunner.kt

```diff
@@ -45,6 +45,9 @@ class SendCoinsTaskRunner @Inject constructor(
 ) : SendPaymentService {
     private val log = LoggerFactory.getLogger(SendCoinsTaskRunner::class.java)
 
+    @JvmField
+    var dryrunException: Exception? = null
```

Comment:
This doesn't look like it's used. This class should not keep a state in any case.